apps.json

```diff
@@ -1020,7 +1020,7 @@
     },
     "ffsync": {
         "category": "synchronization",
-        "level": 0,
+        "level": 6,
```

Comment:
wait wat

Reply:
🧟

Reply:
Ah, @Josue-T wants to keep it alive despite https://github.com/YunoHost-Apps/ffsync_ynh/pull/40

Reply:
![Uploading watdidijustread.gif…]()

Reply:
I did worse, by resurrecting helper `ynh_install_python` at https://github.com/YunoHost-Apps/mediadrop_ynh making mediadrop python2.7 working on bullseye

I will try to make coin level7 on bullseye ^^ but introduced a subject for next meeting https://pad.lqdn.fr/p/yunohost-06-09-2022 `what about using to ynh_install_python ?`